From 57f50e4ddb865c4a3e94251d4a9230526fdc254a Mon Sep 17 00:00:00 2001
From: Matus Ujhelyi <matus.ujhelyi@streamunlimited.com>
Date: Wed, 8 Jan 2014 14:58:05 +0100
Subject: [PATCH] partitions: increase size of settings partition on 4GBit
 flash version

On the 4GBit boards, the small settings partition caused an error
when trying to create the ubi volume.
In addition, it makes more sense to increase the size of the settings
partition instead of the rootfs, because there is no profit in more
unused space on the rootfs.

Signed-off-by: Matus Ujhelyi <matus.ujhelyi@streamunlimited.com>
Signed-off-by: Bernhard Miller <bernhard.miller@streamunlimited.com>
---
 board/streamunlimited/stream800/partitions.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/board/streamunlimited/stream800/partitions.h b/board/streamunlimited/stream800/partitions.h
index ce4a11b..48f54ee 100644
--- a/board/streamunlimited/stream800/partitions.h
+++ b/board/streamunlimited/stream800/partitions.h
@@ -54,10 +54,10 @@ struct partition s800_part_512M[] = {
 	 * and CONFIG_CONSTANTS_BASE  in stream800.h
 	 */
 	[6] = { 0x00300000, 0x00080000, "constants" },
-	[7] = { 0x00380000, 0x00800000, "settings" },
-	[8] = { 0x00B80000, 0x00600000, "kernel" },
-	[9] = { 0x01180000, 0x00200000, "dts" },
-	[10] = { 0x01380000, 0x17300000, "rootfs" },
+	[7] = { 0x00380000, 0x10000000, "settings" },
+	[8] = { 0x10380000, 0x00600000, "kernel" },
+	[9] = { 0x10980000, 0x00200000, "dts" },
+	[10] = { 0x10B80000, 0x07B00000, "rootfs" },
 	[11] = { 0x18680000, 0x07980000, "download" }
 };
 
-- 
2.7.4

